test/primitives-test.coffee

```diff
@@ -146,6 +146,129 @@ describe 'Minim Primitives', ->
           foo: 'bar'
           z: 2
 
+  describe 'ElementType', ->
+    elType = undefined
+
+    before ->
+      elType = new minim.ElementType 'element'
+
+    describe '#getProperty', ->
```

Comment:
Will DRY later, as I think I will be refactoring this in preparation for other element types.